tools/measure_coverage.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+# Copyright 2017-present Samsung Electronics Co., Ltd. and other contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ "$#" -gt "0" ] && ( [ "$1" == "-h" ] || [ "$1" == "--help" ] ); then
+    echo "Measure JavaScript and C coverage and create a html report"
+    echo "out of the results"
+    echo ""
+    echo "Usage: $0 [NODE_MODULES_DIR]"
```

Comment:
This path is not required, it is optional. Basically we assume that it is in main iotjs folder.